tests/testcases/integration.py

```diff
@@ -452,6 +452,26 @@ def test_set_delete_original_true(self):
         self.assertTrue(gio_settings.get_boolean('delete-original'))
         self.assertFalse(os.path.isfile('tests/tmp/a.wav'))
 
+    def test_set_output_resample(self):
+        gio_settings = get_gio_settings()
+        self.assertFalse(gio_settings.get_boolean('output-resample'))
+        self.assertEqual(48000, gio_settings.get_int('resample-rate'))
+
+        os.system('cp "tests/test data/audio/a.wav" "tests/tmp/a.wav"')
+        self.assertTrue(os.path.isfile('tests/tmp/a.wav'))
+
+        launch([
+            '-b', 'tests/tmp/a.wav',
+            '-o', 'tests/tmp',
+            '-f', 'm4a',
+            '-R', '8000'
+        ])
+
+        gio_settings = get_gio_settings()
+        self.assertTrue(gio_settings.get_boolean('output-resample'))
+        self.assertEqual(gio_settings.get_int('resample-rate'), 8000)
+        self.assertEqual(self.get_bitrate('tests/tmp/a.m4a'), 8000)
```

Comment:
This test is failing. I'm not sure how `get_bitrate()` computes bitrate.

Reply:
should be close to file-size / duration.

if you use wav in this test, the bitrate might be quite close to the samplerate.